### ISSUE TITLE
[Docs Request] InceptionV3 Classify multiple images in same run.

### ISSUE THREAD

**Issue by Waffleboy** [tensorflow/tensorflow#3107]
Dearest github community,

Thank you for your wonderful contributions to Tensorflow! I greatly appreciate Tensorflow being open-source, for anyone to use :)

I was browsing the docs. and was attempting to retrain the last layer of the inceptionV3 model on new categories.(From https://www.tensorflow.org/versions/r0.9/how_tos/image_retraining/index.html)

It was successful, and i can predict a single image by running:
```

bazel build tensorflow/examples/label_image:label_image && \
bazel-bin/tensorflow/examples/label_image/label_image \
--graph=/tmp/output_graph.pb --labels=/tmp/output_labels.txt \
--output_layer=final_result \
--image=PATH_TO_IMG
```

However, after much searching, I still could not find a way to use this retrained model to classify multiple images. The link provided here: https://github.com/eldor4do/Tensorflow-Examples/blob/master/retraining-example.py also discusses using the model on a per-image basis.

I've attempted to modify that file to accept multiple images, but I was unsuccessful. (Playing around with feed_dict threw errors.)

It would be really helpful if there was a documentation of some sort to provide guidelines / show how to accomplish this task :) 

Tldr: requesting documentation to show how to classify multiple images using InceptionV3 (tensorflow) in python instead of one by one.

Thank you, and have a great day!

**Comment by andydavis1** [comment]
This is a question better suited for StackOverflow. Please ask it there and tag it with the `tensorflow` tag
